metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -1110,5 +1111,7 @@ def _get_ge_dataset(
 def _get_column_types_to_ignore(dialect_name: str) -> List[str]:
     if dialect_name.lower() == "postgresql":
         return ["JSON"]
+    elif dialect_name.lower() == "trino" and sys.version_info <= (3, 7):
```

Comment:
I think the issue is actually sqlalchemy 1.3 vs 1.4

it just happens that we get 1.3 in ci with python 3.7, and 1.4 in ci with python 3.10